docs/stdlib/sequence.rst

```diff
@@ -66,12 +66,12 @@ Sequences
                   std::sequence_reset( \
                     seq: schema::ScalarType, val: int64) -> int64
 
-    Reset the sequence to its initial state or the specified value.
+    Returns the initial state of a sequence or a specified ``val``.
```

Comment:
This should really be described as a "reset" in that the internal counter of the sequence is reset back to zero. In other words, this has side effects